packages/core/test/tabs/tabs2Tests.tsx

```diff
@@ -206,6 +206,24 @@ describe("<Tabs2>", () => {
         assert.equal(wrapper.find(".pt-tab-indicator").length, 0);
     });
 
+    it("removes indicator element when selected tab is removed", () => {
+        const wrapper = mount(<Tabs2 id={ID}>{getTabsContents()}</Tabs2>);
+        // first tab is selected by default. now remove it.
+        const tabIdsWithoutFirstTab = TAB_IDS.slice(1);
+        wrapper.setProps({ children: getTabsContents(tabIdsWithoutFirstTab) });
+        const indicatorStyle = wrapper.state().indicatorWrapperStyle;
+        assert.deepEqual(indicatorStyle, { display: "none" }, "indicator should be hidden");
+    });
+
+    it("leaves indicator element in place when non-selected tab is removed", () => {
+        const wrapper = mount(<Tabs2 id={ID}>{getTabsContents()}</Tabs2>);
+        // first tab is selected by default. now remove the last one.
+        const lastTabIndex = TAB_IDS.length - 1;
+        const tabIdsWithoutLastTab = TAB_IDS.slice(0, lastTabIndex - 1);
```

Comment:
`.slice(0, -1)` 😮

Reply:
That's so python-y, I can't even.

Noted for next time. 👍